clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/CloudMetricStatistics.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import groovy.transform.Immutable
+
+interface CloudMetricStatistics {
+  /**
+   * Unit of measurement for all datapoints; should be the plural form of the unit if applicable, e.g. "bytes"
+   */
+  String unit
```

Comment:
Is this just a display name, or are these standardized to have semantic meaning? Since you say plural form, I suspect these are just display names.